### PR TITLE
[hi] Localize kubernetes.md to Hindi

### LINKED ISSUE #1588: [hi] Localize kubernetes.md to Hindi
content: [https://glossary.cncf.io/kubernetes/](https://glossary.cncf.io/kubernetes/) 

Please complete DCO requirements before making a PR or commits : [https://wiki.linuxfoundation.org/dco](https://wiki.linuxfoundation.org/dco)

### PR DESCRIPTION
**Pull request by Omkar0114**
Signed-off-by: Omkar kulkarni <88308267+Omkar0114@users.noreply.github.com>

Localize kubernetes.md to Hindi

resolve #1588

- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by Omkar0114** [comment]
Hey, @Garima-Negi , @jayesh-srivastava , @sayantani11 @anubha-v-ardhan can you take a look on this PR? And suggest any changes !

File: content/hi/kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: कुबेरनेट्स(Kubernetes)
+status: Completed
+category: प्रौद्योगिकी
+---
```
Comment (anubha-v-ardhan) [inline content/hi/kubernetes.md:5]:
```suggestion
---
title: कुबेरनेट्स (Kubernetes)
status: Completed
category: प्रौद्योगिकी
tags: ["infrastructure", "fundamental", ""]
---
```

File: content/hi/kubernetes.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+कुबेरनेट्स, जिसे अक्सर K8s के रूप में संक्षिप्त किया जाता है, एक ओपन सोर्स कंटेनर ऑर्केस्ट्रेटर है।
+यह एक "डेटासेंटर ऑपरेटिंग सिस्टम" के रूप में कार्य करते हुए आधुनिक बुनियादी ढांचे पर कंटेनरीकृत अनुप्रयोगों के जीवनचक्र को स्वचालित करता है जो एक [वितरित प्रणाली](/distributed-systems/) में अनुप्रयोगों का प्रबंधन करता है।
+
+कुबेरनेट्स [कंटेनर](/container/) [नोड्स](/nodes/) में एक [क्लस्टर](/cluster/) में शेड्युल करता है। कंटेनरीकृत अनुप्रयोगों को चलाने के लिए लोड बैलेंसर, लगातार भंडारण, आदि जैसे कई बुनियादी ढांचे के संसाधनों को बंडल करता है।
+
+कुबेरनेट्स स्वचालन और विस्तारणीयता को सक्षम करता है, जिससे उपयोगकर्ता पुनरुत्पादित तरीके से घोषणात्मक रूप से (नीचे देखें) अनुप्रयोगों को तैनात कर सकते हैं।
+Kubernetes अपने [API](/application-programming-interface/) के माध्यम से एक्स्टेंसिबल है, अनुभवी कुबेरनेट्स चिकित्सकों को उनकी आवश्यकताओं के अनुसार इसकी स्वचालन क्षमताओं का लाभ उठाने की अनुमति देता है।
+
+## समस्या जिसे यह संबोधित करता है
```
Comment (anubha-v-ardhan) [inline content/hi/kubernetes.md:17]:
```suggestion
## समस्या 
```

File: content/hi/kubernetes.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+कुबेरनेट्स स्वचालन और विस्तारणीयता को सक्षम करता है, जिससे उपयोगकर्ता पुनरुत्पादित तरीके से घोषणात्मक रूप से (नीचे देखें) अनुप्रयोगों को तैनात कर सकते हैं।
+Kubernetes अपने [API](/application-programming-interface/) के माध्यम से एक्स्टेंसिबल है, अनुभवी कुबेरनेट्स चिकित्सकों को उनकी आवश्यकताओं के अनुसार इसकी स्वचालन क्षमताओं का लाभ उठाने की अनुमति देता है।
+
+## समस्या जिसे यह संबोधित करता है 
+
+इन्फ्रास्ट्रक्चर ऑटोमेशन और घोषणात्मक कॉन्फ़िगरेशन प्रबंधन लंबे समय से महत्वपूर्ण अवधारणाएं रही हैं, लेकिन [क्लाउड कंप्यूटिंग](/cloud-computing/) ने लोकप्रियता हासिल की है, इसलिए वे अधिक महत्वपूर्ण हो गए हैं।
+जैसे-जैसे कम्प्यूट संसाधनों की मांग बढ़ती है और संगठनों को कम इंजीनियरों के साथ अधिक परिचालन क्षमता प्रदान करने की आवश्यकता होती है, उस मांग को पूरा करने के लिए नई तकनीकों और कार्य विधियों की आवश्यकता होती है।
+
+## यह कैसे मदद करता है
```
Comment (anubha-v-ardhan) [inline content/hi/kubernetes.md:22]:
```suggestion
## समाधान
```

File: content/hi/kubernetes.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+---
+title: कुबेरनेट्स(Kubernetes)
+status: Completed
+category: प्रौद्योगिकी
+---
+
+## यह क्या है
+
+कुबेरनेट्स, जिसे अक्सर K8s के रूप में संक्षिप्त किया जाता है, एक ओपन सोर्स कंटेनर ऑर्केस्ट्रेटर है।
+यह एक "डेटासेंटर ऑपरेटिंग सिस्टम" के रूप में कार्य करते हुए आधुनिक बुनियादी ढांचे पर कंटेनरीकृत अनुप्रयोगों के जीवनचक्र को स्वचालित करता है जो एक [वितरित प्रणाली](/distributed-systems/) में अनुप्रयोगों का प्रबंधन करता है।
```
Comment (anubha-v-ardhan) [inline content/hi/kubernetes.md:10]:
```suggestion
यह एक "डेटासेंटर ऑपरेटिंग सिस्टम" के रूप में कार्य करते हुए आधुनिक बुनियादी ढांचे पर कंटेनरीकृत एप्लीकेशन के जीवनचक्र को स्वचालित करता है जो एक [वितरित प्रणाली](/distributed-systems/) में एप्लीकेशन का प्रबंधन करता है।
```

File: content/hi/kubernetes.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+कुबेरनेट्स [कंटेनर](/container/) [नोड्स](/nodes/) में एक [क्लस्टर](/cluster/) में शेड्युल करता है। कंटेनरीकृत अनुप्रयोगों को चलाने के लिए लोड बैलेंसर, लगातार भंडारण, आदि जैसे कई बुनियादी ढांचे के संसाधनों को बंडल करता है।
```
Comment (anubha-v-ardhan) [inline content/hi/kubernetes.md:12]:
```suggestion
कुबेरनेट्स [कंटेनर](/container/) को [नोड्स](/nodes/) में एक [क्लस्टर](/cluster/) में शेड्युल करता है। कंटेनरीकृत एप्लीकेशन को चलाने के लिए लोड बैलेंसर, आग्रही (persistent) स्टोरेज, आदि जैसे कई बुनियादी ढांचे के संसाधनों को बंडल करता है।
```

File: content/hi/kubernetes.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+कुबेरनेट्स स्वचालन और विस्तारणीयता को सक्षम करता है, जिससे उपयोगकर्ता पुनरुत्पादित तरीके से घोषणात्मक रूप से (नीचे देखें) अनुप्रयोगों को तैनात कर सकते हैं।
```
Comment (anubha-v-ardhan) [inline content/hi/kubernetes.md:14]:
```suggestion
कुबेरनेट्स स्वचालन और विस्तारणीयता को सक्षम करता है, जिससे उपयोगकर्ता पुनरुत्पादित तरीके से घोषणात्मक रूप से (नीचे देखें) एप्लीकेशन को तैनात कर सकते हैं।
```

File: content/hi/kubernetes.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+## यह कैसे मदद करता है
+
+पारंपरिक [इन्फ्रास्ट्रक्चर ऐज कोड](/infrastructure-as-code/) टूल्स के समान, कुबेरनेट्स स्वचालन के साथ मदद करता है लेकिन कंटेनरों के साथ काम करने का फायदा है।
+[आभासी](/virtual-machine/) या भौतिक मशीनों की तुलना में कंटेनर कॉन्फ़िगरेशन बहाव के प्रति अधिक प्रतिरोधी होते हैं।
+
+इसके अतिरिक्त, कुबेरनेट्स घोषणात्मक रूप से काम करता है, जिसका अर्थ है कि ऑपरेटरों को मशीन को निर्देश देने के बजाय कि कुछ कैसे करना है, वे वर्णन करते हैं - आम तौर पर मेनिफेस्ट फाइलों के रूप में (उदाहरण के लिए, YAML) - बुनियादी ढांचा कैसा दिखना चाहिए।
+कुबेरनेट्स तब "कैसे" का ख्याल रखता है।
+इसके परिणामस्वरूप कुबेरनेट्स बुनियादी ढांचे के साथ कोड के रूप में बेहद संगत है।
+
+कुबेरनेट्स [स्वयं](/सेल्फ-हीलिंग/) ठीक हो जाता है।
```
Comment (anubha-v-ardhan) [inline content/hi/kubernetes.md:31]:
```suggestion
कुबेरनेट्स [स्वयं](/self-healing/) ठीक हो जाता है।
```

File: content/hi/kubernetes.md (hunk 1 above)
Comment (Omkar0114) [reply content/hi/kubernetes.md]: Updated :)

File: content/hi/kubernetes.md (hunk 3 above)
Comment (Omkar0114) [reply content/hi/kubernetes.md]: updated

File: content/hi/kubernetes.md (hunk 5 above)
Comment (Omkar0114) [reply content/hi/kubernetes.md]: Updated :+1:

File: content/hi/kubernetes.md (hunk 7 above)
Comment (Omkar0114) [reply content/hi/kubernetes.md]: updated

**Comment by Omkar0114** [comment]
Hi, @anubha-v-ardhan Updated docs with your suggestions, have a look once. Suggest any changes if needed :)

**Review by anubha-v-ardhan — APPROVED** [review]
LGTM!

Please wait for one more approval from another maintainer before we can merge this.

**Comment by Omkar0114** [comment]
Hi, one approved is there! Anyone can review it @Garima-Negi @sayantani11 @jayesh-srivastava

**Comment by Omkar0114** [comment]
Hey @anubha-v-ardhan @sayantani11 @Garima-Negi @jayesh-srivastava Anyone can review this PR as one LGTM has been added!

File: content/hi/kubernetes.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+title: कुबेरनेट्स (Kubernetes)
+status: Completed
+category: प्रौद्योगिकी
+tags: ["infrastructure", "fundamental", ""]
+---
+
+## यह क्या है
+
+कुबेरनेट्स, जिसे अक्सर K8s के रूप में संक्षिप्त किया जाता है, एक ओपन सोर्स कंटेनर ऑर्केस्ट्रेटर है।
+यह एक "डेटासेंटर ऑपरेटिंग सिस्टम" के रूप में कार्य करते हुए आधुनिक बुनियादी ढांचे पर कंटेनरीकृत एप्लीकेशन के जीवनचक्र को स्वचालित करता है जो एक [वितरित प्रणाली](/distributed-systems/) में एप्लीकेशन का प्रबंधन करता है।
```
Comment (aj11anuj) [inline content/hi/kubernetes.md:11]:
```suggestion
यह एक "डेटासेंटर ऑपरेटिंग सिस्टम" के रूप में कार्य करते हुए आधुनिक इन्फ्रा़स्ट्रक्चर पर कंटेनरीकृत एप्लीकेशन के जीवनचक्र को स्वचालित करता है जो एक [वितरित प्रणाली](/distributed-systems/) में एप्लीकेशन का प्रबंधन करता है।
```

File: content/hi/kubernetes.md [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+
+कुबेरनेट्स [कंटेनर](/container/) को [नोड्स](/nodes/) में एक [क्लस्टर](/cluster/) में शेड्युल करता है। कंटेनरीकृत एप्लीकेशन को चलाने के लिए लोड बैलेंसर, आग्रही (persistent) स्टोरेज, आदि जैसे कई बुनियादी ढांचे के संसाधनों को बंडल करता है।
```
Comment (aj11anuj) [inline content/hi/kubernetes.md:13]:
```suggestion
कुबेरनेट्स [कंटेनर](/container/) को [नोड्स](/nodes/) के एक [क्लस्टर](/cluster/) में शेड्युल करता है। कंटेनरीकृत एप्लीकेशन को चलाने के लिए लोड बैलेंसर, सतत (persistent) स्टोरेज, आदि जैसे कई इन्फ्रा़स्ट्रक्चर के संसाधनों को बंडल करता है।
```

File: content/hi/kubernetes.md [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+
+कुबेरनेट्स स्वचालन और विस्तारणीयता को सक्षम करता है, जिससे उपयोगकर्ता पुनरुत्पादित तरीके से घोषणात्मक रूप से (नीचे देखें) एप्लीकेशन को तैनात कर सकते हैं।
+Kubernetes अपने [API](/application-programming-interface/) के माध्यम से एक्स्टेंसिबल है, अनुभवी कुबेरनेट्स चिकित्सकों को उनकी आवश्यकताओं के अनुसार इसकी स्वचालन क्षमताओं का लाभ उठाने की अनुमति देता है।
+
+## समस्या
+
+इन्फ्रास्ट्रक्चर ऑटोमेशन और घोषणात्मक कॉन्फ़िगरेशन प्रबंधन लंबे समय से महत्वपूर्ण अवधारणाएं रही हैं, लेकिन [क्लाउड कंप्यूटिंग](/cloud-computing/) ने लोकप्रियता हासिल की है, इसलिए वे अधिक महत्वपूर्ण हो गए हैं।
+जैसे-जैसे कम्प्यूट संसाधनों की मांग बढ़ती है और संगठनों को कम इंजीनियरों के साथ अधिक परिचालन क्षमता प्रदान करने की आवश्यकता होती है, उस मांग को पूरा करने के लिए नई तकनीकों और कार्य विधियों की आवश्यकता होती है।
```
Comment (aj11anuj) [inline content/hi/kubernetes.md:21]:
```suggestion
जैसे-जैसे कम्प्यूट संसाधनों की मांग बढ़ती है और संगठनों को कम इंजीनियरों के साथ अधिक आयोजन करने की क्षमता प्रदान करने की आवश्यकता होती है, उस मांग को पूरा करने के लिए नई तकनीकों और कार्य विधियों की आवश्यकता होती है।
```

**Review by aj11anuj — CHANGES_REQUESTED** [review]
Some suggestions from my side

**Review by aj11anuj — APPROVED** [review]
Lgtm 👍

**Review by kumarankit999 — APPROVED** [review]
lgtm

**Pull request merged**